smartapps/alarmdecoder/alarmdecoder-service.src/alarmdecoder-service.groovy

```diff
@@ -1986,6 +1986,43 @@ def rfxSet(evt) {
   def device_name = "RFX-${sn}-${bat}-${supv}-" +
     "${loop0}-${loop1}-${loop2}-${loop3}"
 
+    def d = getChildDevices().findAll {
+			it.deviceNetworkId.contains(":switch") &&
+			  it.getDataValue("serial") == sn
+		  }
+		  
+		  if (d) {
+			if (loop1 == "1" || loop2 == "1" || loop3 == "1" || loop4 == "1") {
+				d.each {
+				  _sendEventTranslate(it, ("on"), false)
+				}
+			}
+			else {
+				d.each {
+				  _sendEventTranslate(it, ("off"), false)
+				}
+				
+				d.each {
+					it.sendEvent(
+					  name: "low_battery",
+					  value: bat == "1"
+					)
+				}
+				
+				if (supv == "1") {
+					def last_checkin = now()
+					d.each {
+						it.sendEvent(
+						  name: "last_checkin",
+						  value: last_checkin
+						)
+					}
+				}
+			}
+			 
+		  } 
+
+    
```

Comment:
This section needs to have it's whitespace cleaned up so that it matches the rest of the file. (2 space tabstops)